infrastructure/src/main/java/org/corfudb/infrastructure/FailureReportModel.java

```diff
@@ -0,0 +1,33 @@
+package org.corfudb.infrastructure;
+
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Failure Report Model:
```

Comment:
This comment is not very informative. Not sure what this is for. It is hard to get the picture without comments describing the intention of a class.